src/Resources/app/administration/package.json

```diff
@@ -7,7 +7,8 @@
   "license": "MIT",
   "scripts": {
     "lint": "eslint --ext .js,.ts,.vue,.html,.html.twig .",
-    "lint-fix": "npm run lint -- --fix"
+    "lint-fix": "npm run lint -- --fix",
+    "lint:css": "stylelint ./**/*.scss"
```

Comment:
Can you please also add `lint:css` to the general `lint` script? Because then it would also be included in our CI pipeline. The alternative would be to create an additional CI job for it.

I think something like this should work:
```
"lint": [
    "eslint --ext .js,.ts,.vue,.html,.html.twig .",
    "npm run lint:css",
],
```

Reply:
Ok, I've added a new job.